src/inspected-window-helpers/overlay-helpers.js

```diff
@@ -35,28 +34,25 @@ export function setupOverlayHelpers() {
   // everything after this are helper functions
 
   function getSelectorsAndOptions(app) {
+    const { selectors } = app.devtools.overlays;
+    if (!selectors.length) {
+      selectors.push(`#single-spa-application\\:${app.name}`);
+    }
     return {
-      selectors: app.devtools.overlays.selectors
-        .map(selector => {
-          if (document.querySelector(selector)) {
-            return selector;
-          } else {
-            return undefined;
-          }
-        })
-        .filter(selection => selection),
+      selectors: selectors
```

Comment:
maybe rename `selectors` to `elements`? It confused me at first reading code that calls `getSelectorsAndOptions` but the selectors variable is a dom element instead of a selector.

Reply:
Yea I was trying to make as few changes as needed but should've just done this. Will change.